### DISCUSSION TITLE
Subscribe

### DISCUSSION THREAD

**Discussion by SinkoOfficial** [community/community#146419] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was charged a day for a subscription that I forgot about, but I don't use it, I created a ticket and was told that the funds were returned, but a day has already passed and the money has not come

**Comment by mecodeatlas** [comment]
Hey, @SinkoOfficial! thanks for reaching out! 
 
It’s very likely that the refund process takes more than 24 hours. In this case, we recommend contacting the [support team](https://support.github.com/contact) to confirm the exact timeframe for receiving your refund. They will surely be able to assist you!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
